modules/creating-a-connection-type.adoc

```diff
@@ -0,0 +1,61 @@
+:_module-type: PROCEDURE
+
+[id="creating-a-connection-type_{context}"]
+= Creating a connection type
+
+[role='_abstract']
+As an {productname-short} administrator, you can create a connection type for users in your organization.
+
+ifdef::upstream[]
+You can create a new connection type as described in this procedure or you can create a copy of an existing connection type and edit it, as described 
+in link:{odhdocshome}/managing-odh/#duplicating-a-connection-type_resource-mgmt[Duplicating a connection type].
+endif::[]
+ifndef::upstream[]
+You can create a new connection type as described in this procedure or you can create a copy of an existing connection type and edit it, as described in link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/managing-connection-types#duplicating-a-connection-type_resource-mgmt[Duplicating a connection type].
+endif::[]
+
+.Prerequisites
+* You have logged in to {productname-short} as a user with {productname-short} administrator privileges. 
```

Comment:
Is this really the only prerequisite required to create a connection type? I'd have assumed there would be more.